doc/API/centreon-api-v2.yaml

```
@@ -1415,6 +1450,137 @@ components:
               description: "ID of the service on which the downtime is set"
               example : 5
+    Monitoring.ResourceMain:
+      type: object
+      properties:
+        type:
+          type: string
+          description: "Type of the resource"
+          example: "service"
+        id:
+          type: integer
+          format: int64
+          description: "ID of the resource"
+          example: 12
+        name:
+          type: string
+          description: "Resource name"
+          example: "Ping"
+        details_url:
+          type: string
+          description: "Resource url to details page"
+          example: "/monitoring/host/12/service/12"
+        icon:
+          type: object
+          properties:
+            name:
+              type: string
+              description: "Name of the icon"
+              example: "memory"
+            url:
+              type: string
+              description: "Url of the icon"
+              example: "/media/memory.png"
+        parent:
+          type: object
+          properties:
+            id:
+              type: integer
+              format: int64
+              description: "ID of the parent resource"
+              example: 12
+            name:
+              type: string
+              description: "Parent resource name"
+              example: "Central"
+            details_url:
+              type: string
+              description: "Parent resource url to details page"
+              example: "/monitoring/host/12"
+            icon:
+              type: object
+              properties:
+                name:
+                  type: string
+                  description: "Name of the icon"
+                  example: "linux"
+                url:
+                  type: string
+                  description: "Url of the icon"
+                  example: "/media/linux.png"
+        status:
+          type: object
+          properties:
+            code:
+              type: integer
+              description: |
+                resource status code
+                * `0` - UP or OK
+                * `1` - DOWN or WARNING
+                * `2` - UNREACHABLE or CRITICAL
+                * `3` - UNKNOWN
+                * `4` - PENDING
+              example: 0
+            name:
+              type: integer
+              description: |
+                resource status name
+                * UP
+                * DOWN
+                * UNREACHABLE
+                * PENDING
+                * OK
+                * WARNING
+                * CRITICAL
+                * UNKNOWN
+              example: "UP"
+        in_downtime:
+          type: boolean
+          description: "Indicates whether resource is in downtime"
+          example: false
+        acknowledged:
+          type: boolean
+          description: "Indicates whether resource is acknowledged"
+          example: false
+        severity:
+          type: object
+          properties:
```

the object must have an identifier

```suggestion
          properties:
            id:
              type: integer
              format: int64
              description: "ID of the severity"
              example: 12
```